SUMMARY.md

```diff
@@ -9,8 +9,9 @@
 - [🎓 Guides](guides/README.md)
   - [🚶 First Steps in Crypto](guides/newbie.md)
   - [💶 Get agEUR](guides/getAgEUR.md)
-  - [💸 Pay and get paid with agEUR](guides/ageur-power-usecase.md)
-  - [🏦 agEUR as a treasury asset](guides/ageur-treasury.md)
+  - [🧩  Use cases](/guides/use-cases/ageur-power-usecase.md)
```

Comment:
Let's make a README for the use cases section

Reply:
Why? Do you think there is anything useful to write there?

Reply:
No but out of consistency we can just have a small page that says: in this section there are guides about what you can do with agEUR